code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1727,6 +1727,19 @@
 		src.desc = "A wish come true!"
 		reagents.add_reagent(NUTRIMENT, 8)
 
+/obj/item/weapon/reagent_containers/food/snacks/avocadosoup
+	name = "Vegetable soup"
+	desc = "May be served either hot or cold."
+	icon_state = "avocadosoup"
+	trash = /obj/item/trash/snack_bowl
+	food_flags = FOOD_LIQUID
+
+/obj/item/weapon/reagent_containers/food/snacks/avocadosoup/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 8)
```

Comment:
Also nutriment clearly has more to do with how full it makes you feel (meaning how much fat it has) than how nutritious it is
That's why corn oil is better nutriment than nutriment
And avocados are not lacking in fat